app/elements/plans-list/plans-list.html

```diff
@@ -42,19 +42,17 @@
                 is: 'plans-list',
 
                 properties: {
+                    user: {
```

Comment:
This element only needs the list of plans and the current plan as input. No need to pass the whole user object